streams/test-utils/src/main/java/org/apache/kafka/streams/test/OutputVerifier.java

```diff
@@ -352,4 +360,79 @@
         Objects.requireNonNull(expectedRecord);
         compareKeyValueHeaders(record, expectedRecord.key(), expectedRecord.value(), expectedRecord.headers());
     }
+
+    /**
+     * Compares a {@link ProducerRecord} with the provided key, value, and headers and throws an
```

Comment:
`key, value, [timestamp, ] and headers`

`timestamp` is missing on other places in this and the below JavaDoc, too.